charts/fission-all/values.yaml

```diff
@@ -33,6 +33,9 @@ routerPort: 31314
 ## Port at which NATS streaming service should be exposed
 natsStreamingPort: 31316
 
+## Set to false if you create the namespaces manually
+createNamespace: false
```

Comment:
You said it defaults to true, so this should be `true`, right?

Reply:
Good point, will fix. Cheers